### PR TITLE
error instead of warning

### PR DESCRIPTION
**Pull request by SimonDanisch**
...to get the full stacktrace

### CODE REVIEW

File: src/functionloading.jl [hunk 1]
```diff
@@ -19,7 +19,7 @@ function debug_opengl_expr(func_name, args)
             err = glGetError()
             if err != GL_NO_ERROR
                 arguments = gl_represent.(tuple($(args...)))
-                @warn("OpenGL call to $($func_name), with arguments: $(arguments)
+                error("OpenGL call to $($func_name), with arguments: $(arguments)
```
Comment (asinghvi17) [inline src/functionloading.jl]:
```suggestion
                @error("OpenGL call to $($func_name), with arguments: $(arguments)
```

you need the macro for the stacktrace, I think

Comment (SimonDanisch) [reply src/functionloading.jl]: No this just interrupts

Comment (SimonDanisch) [reply src/functionloading.jl]: So you get an error & stacktrace

Comment (asinghvi17) [reply src/functionloading.jl]: perfect, looks like it worked

**Comment by musm** [comment]
Seems logical to merge this

**Pull request merged**